src/cpp/src/speculative_decoding/speculative_decoding_impl.hpp

```diff
@@ -8,33 +8,10 @@
 #include "speculative_decoding/continuous_batching_for_speculative_decoding_impl.hpp"
 #include "speculative_decoding/speculative_decoding_metrics.hpp"
 #include "openvino/genai/speculative_decoding/perf_metrics.hpp"
+#include "utils.hpp"
```

Comment:
are you sure it is the right location?

Reply:
Yes, it is done specifically to relocate `ModelDesc` from this file to `utils.hpp`, as I am using the same `ModelDesc` in Speculative Decoding implementation